metricshub-hardware/src/it/resources/os/SuperConnectorOsIT/connectors/SuperConnectorCommon-header/SuperConnectorCommon-header.yaml

```diff
@@ -72,7 +72,7 @@ monitors:
         source: "${source::monitors.lun.discovery.sources.source(1)}"
         attributes:
           id: $4
-          __display_id: $2
+          __display_id: $index

```

Comment:
Please, change to `__display_id: $2 $index` so that we keep the IT tests on `LunOnline`, `LunUnknown`, `LunOffline` and `LunNoPath`